### ISSUE TITLE
owlcat vs python3-meqtrees-cattery on KERN-7: broken dep?

### ISSUE THREAD

**Issue by o-smirnov** [kernsuite/packaging#246]
```
root@baker:/etc/netplan# apt install owlcat python3-owlcat
Reading package lists... Done
Building dependency tree       
Reading state information... Done
Some packages could not be installed. This may mean that you have
requested an impossible situation or if you are using the unstable
distribution that some required packages have not yet been created
or been moved out of Incoming.
The following information may help to resolve the situation:

The following packages have unmet dependencies:
 python3-owlcat : Depends: python3-meqtrees-cattery but it is not installable
E: Unable to correct problems, you have held broken packages.
root@baker:/etc/netplan# apt install owlcat python3-owlcat python3-meqtrees-cattery
Reading package lists... Done
Building dependency tree       
Reading state information... Done
Package python3-meqtrees-cattery is not available, but is referred to by another package.
This may mean that the package is missing, has been obsoleted, or
is only available from another source

E: Package 'python3-meqtrees-cattery' has no installation candidate
root@baker:/etc/netplan# 
```

**Comment by gijzelaerr** [comment]
owlcat depends on meqtrees cattery:

https://github.com/ska-sa/owlcat/blob/master/setup.py#L12

unfortunately, meqtrees has not been ported to py3 (yet).

if it is an optional dependency, it should be made an optional dependency upstream.

**Comment by o-smirnov** [comment]
Actually, meqtrees-cattery has been ported. In fact the whole stack is ported to py3. The only issue is that the meqbrowser app needs qt4, which has been dropped in Focal.

So we need to make a plan to package everything except the browser (or even package the browser and have it bomb out with an error message).

Cheers,
Oleg

**Comment by gijzelaerr** [comment]
cool cool. @Athanaseus can you try to create a package for https://github.com/ska-sa/meqtrees-cattery/releases/tag/v1.7.2 ?

**Comment by o-smirnov** [comment]
But also ``meqtrees-timba`` please, sans browser.

**Comment by Athanaseus** [comment]
I uploaded `meqtrees-cattery`.  I'm stuck with `meqtrees-timba` since it depends on `casarest`,  and apparently, this is a stale package, I wasn't able to package it. I have couple of issues, some which I solved others not. I have opened the issue upstream but no response yet. https://github.com/casacore/casarest/issues/40.

If I can remember we only need `casarest` for `lwimager`? Can timba roll without `casarest`?

**Comment by o-smirnov** [comment]
Yeah absolutely. I think the only place it uses ``lwimager`` is in the batch test. We could even just redo the test using wsclean?

**Comment by Athanaseus** [comment]
Yes, that's even a better alternative.
Are you referring to the the `make_ms.sh` here https://github.com/ska-sa/meqtrees-cattery/search?q=lwimager?

**Comment by o-smirnov** [comment]
That's just used to insert MODEL_DATA and CORRECTED_DATA columns. Not sure it's even necessary TBH!

**Comment by gijzelaerr** [comment]
what about this one @Athanaseus , has this been resolved or is this still work in progress?

**Comment by Athanaseus** [comment]
This should be resolved. The latest `Meqtrees` and `owlcat` are uploaded to KERN-7.
